src/internal/git/pull.go

```diff
@@ -16,7 +16,10 @@ import (
 const onlineRemoteName = "online-upstream"
 
 func DownloadRepoToTemp(gitUrl string, spinner *message.Spinner) string {
-	path, _ := utils.MakeTempDir(config.CommonOptions.TempDirectory)
+	path, err := utils.MakeTempDir(config.CommonOptions.TempDirectory)
+	if err != nil {
+		message.Fatalf(err, "Specified tmpdir does not exist, please create it: %s", config.CommonOptions.TempDirectory)
```

Comment:
Same as above, there may be other reasons besides non-existence that this may fail.